app/main/views/users.py

```diff
@@ -296,3 +296,32 @@ def check_supplier_role(role, supplier_id):
         abort(400, "'supplier_id' is required for users with 'supplier' role")
     elif role != 'supplier' and supplier_id:
         abort(400, "'supplier_id' is only valid for users with 'supplier' role, not '{}'".format(role))
+
+
+@main.route("/buyers/briefs", methods=["GET"])
```

Comment:
The admin wants all buyers, even if they don't have any briefs so I've gone for adding a role filter to /users and combining in the admin app. I've got a PR up [here](https://github.com/alphagov/digitalmarketplace-api/pull/431) for it so will close this PR.